main.tf

```diff
@@ -101,6 +116,10 @@ resource "aws_lambda_function" "security_reporter_lambda" {
       ACCOUNT_ID                  = data.aws_caller_identity.current.account_id
       ACCOUNT_ALIAS               = data.aws_iam_account_alias.current.account_alias
       PUBLISH_OK_MESSAGE_TO_SLACK = var.publish_ok_message_to_slack
+      PS_KEY_CONTROLS_IDS_API_HOST     = local.ps_key_security_controls_api_host
+      PS_KEY_CONTROLS_IDS_API_KEY     = local.ps_key_security_controls_api_key
+      PS_KEY_CONTROLS_IDS_API_RESOURCE_PATH     = local.ps_key_security_controls_api_resource_path
```

Comment:
I do not think that having:
`ps_key_security_controls_api_host` and `ps_key_security_controls_api_resource_path`
is need in parameter store but I am ok with that.